ajax/libs/d3-transition/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "d3-transition",
+  "filename": "d3-transition.min.js",
+  "description": "Animated transitions for D3 selections.",
+  "keywords": [
+    "d3",
+    "d3-module",
+    "dom",
+    "transition",
+    "animation"
+  ],
+  "homepage": "https://d3js.org/d3-transition/",
+  "license": "BSD-3-Clause",
+  "author": {
+    "name": "Mike Bostock",
+    "url": "https://bost.ocks.org/mike"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/d3/d3-transition.git"
+  },
+  "npmName": "d3-transition",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/*"
```

Comment:
Please use `**/!(package.js|version.js)` here, package.js appeared in v0.2.9/v0.2.10 and version.js appeared in v0.2.6~v0.2.8.